user-profiles.md

```diff
@@ -0,0 +1,49 @@
+# User Profiles
+
+The purpose of this document is to aid in the development of Tekton features by evaluating Tekton features against those who will be using them.
+
+When considering requirements and implementation solutions it's useful look at who uses Tekton to have a better understanding of how something should work. The Tekton user profiles describe the different types of users and contributors to Tekton along with the order of priority they have relative to each other. The ordering is because nothing can share the exact same priority and we want to convey that.
+
+## Preface
+
+* What's described here are user profiles as opposed to [personas](https://en.wikipedia.org/wiki/Persona#In_user_experience_design). Personas are example actors rather than general categories.
+* Kubernetes Cluster Operators are out of scope for this document. A cluster operator is one who manages the operation of a Kubernetes cluster that applications can run in.
+* Application Operators are in scope for this document at a lower priority than Pipeline Operators. An application operator is one who manages the operation of the application in a Kubernetes cluster.
+
+## Profiles
+
+Profiles describe a type of role a user may perform. A real person may perform more than one role and have more than one profile apply to them. How this mapping works between profiles and real people can vary between companies and other organizations. To handle this variation we focus on the user profiles rather than how they may map to people in these different organizations.
+
+### 1. Pipeline Operator
+
+Pipeline operators define, schedule, monitor, and otherwise operate pipelines across one or more Kubernetes clusters. For example, the operation of a set of Kaniko build pipelines and Argo CD pipelines. This is not to be confused with the role of a Kubernetes cluster operator. This is also not to be confused with the application's operator role, who would oversee a product in production.
+
+### 2. Pipeline Distributor
+
+Distributors are people who package an application and application pipelines for someone else to operate. Examples of this would be those who maintain [Catalog Pipelines](https://github.com/tektoncd/catalog) such as for Kaniko and ArgoCD.
```

Comment:
Minor nit: The catalog doesn't includes full Pipelines right now.  I suggest changing this to "Catalog Tasks" instead of "Catalog Pipelines" to be more strictly correct about the current state.

Reply:
Will the catalog allow Pipelines, @vdemeester ? :-)

:ok:

Reply:
If it will allow it soon/later, then *great*. If the entire scope is intended for "parts of a pipeline" or some other adequate description, I can place it here now, while we have it under discussion.

Will replace with "Catalog Tasks" in the next revision, unless suggested another better descriptive text

Reply:
"Examples of this would be those who maintain [Tekton's Catalog](https://github.com/tektoncd/catalog) for Kaniko and ArgoCD."

Dropped the noun specifying what is found in the Catalog, for minor future-proofing against the other 2-3 types of objects intended for Catalog